tools/app/app/api/cardTypes/route.tsx

```diff
@@ -17,15 +17,15 @@ export const dynamic = 'force-dynamic';
 
 /**
  * @swagger
- * /api/cardtypes:
+ * /api/cardTypes:
```

Comment:
Do you think it's beneficial to also use camelCase for routes? I've never seen that. I think most apis use lowercase / underscores.

Reply:
If they work, then why not?

Reply:
We are already looking a bit `weird` in the Linux filesystem with `cardRoot` and `cardTypes`. More natural for Linux filesystem folder/file names would be `card-root` and `card-types`.